src/main/java/nl/tudelft/ti2206/group9/util/SaveGame.java

```diff
@@ -0,0 +1,60 @@
+package nl.tudelft.ti2206.group9.util;
+
+import nl.tudelft.ti2206.group9.level.State;
+
+import java.io.File;
+
+/**
+ * This class takes care of all the saving and loading of the game.
+ * @author Mathias
+ */
+public final class SaveGame {
+
+    /** Private constructor. */
+    private SaveGame() { }
+
+    /**
+     * Delegate the loading of the game to Parser class.
+     * @param fileName the file to be loaded
+     */
+    public static void loadGame(final String fileName) {
+        final String filePath = State.getDefaultSaveDir() + fileName + ".json";
+        Parser.loadGame(filePath);
+    }
+
+    /**
+     * Delegate the saving of the game to Writer class.
+     */
+    public static void saveGame() {
+        final String filePath = State.getDefaultSaveDir()
+                + State.getPlayerName() + ".json";
+        Writer.saveGame(filePath);
+    }
+
+    /**
+     * Read all names of the files in the default savegame directory
+     * and store them in ObservableList players.
+     */
+    public static void readPlayerNames() {
+        final File folder = new File(State.getDefaultSaveDir());
+        for (final File file : folder.listFiles()) {
```

Comment:
Findbugs doesn't like this statment, as it could trigger a nullpointer.
I quote: "Possible null pointer dereference in nl.tudelft.ti2206.group9.util.SaveGame.readPlayerNames() due to return value of called method".
Maybe you can check it for being null.

<review,bug